test/unit/app/common/lib/ledgerExportUtilTest.js

```diff
@@ -7,20 +7,14 @@ const moment = require('moment')
 require('../../../braveUnit')
 
 const ledgerExportUtil = require('../../../../../app/common/lib/ledgerExportUtil')
-const base64Encode = require('../../../../../js/lib/base64').encode
 
-const CSV_HEADER_ROW_PREFIX_COLUMNS = ['Publisher', 'Votes', 'Fraction', 'BTC']
+const CSV_HEADER_ROW_PREFIX_COLUMNS = ['Publisher', 'Votes', 'Fraction']
 const CSV_HEADER_ROW_PREFIX = CSV_HEADER_ROW_PREFIX_COLUMNS.join(',')
 const DEFAULT_CSV_HEADER_ROW_COLUMNS = CSV_HEADER_ROW_PREFIX_COLUMNS.concat(['USD'])
 const DEFAULT_CSV_HEADER_ROW = DEFAULT_CSV_HEADER_ROW_COLUMNS.join(',')
 const CSV_COLUMN_COUNT = DEFAULT_CSV_HEADER_ROW.split(',').length
-const EMPTY_CSV = DEFAULT_CSV_HEADER_ROW
 // N.B. the expected datatype for the 'USD'/fiat column is a 'string' because it is of form '5.00 USD'
-const CSV_EXPECTED_COLUMN_DATATYPES = ['string', 'number', 'number', 'number', 'string']
-
-const CSV_CONTENT_TYPE = 'text/csv'
-const CSV_DATA_URI_PREFIX = 'data:' + CSV_CONTENT_TYPE + ';base64,'
-const EMPTY_CSV_DATA_URL = CSV_DATA_URI_PREFIX + base64Encode(EMPTY_CSV)
+const CSV_EXPECTED_COLUMN_DATATYPES = ['string', 'number', 'number', 'string']
```

Comment:
since there are now only 3 columns, do we need to remove the last from here? (since this has 4 entries)

Reply:
we add forth (fiat) column dynamically

Reply:
👍